wrapper/xgboost.py

```diff
@@ -478,10 +490,16 @@ def load_model(self, fname):
 
         Parameters
         ----------
-        fname : string
-            Input file name.
+        fname : string of file handle
+            Input file name or file handle object.
         """
-        xglib.XGBoosterLoadModel(self.handle, c_str(fname))
+        if isinstance(fname, string_types):  # assume file name
+            xglib.XGBoosterLoadModel(self.handle, c_str(fname))
+        else:
+            buf = fname.getbuffer()
+            length = ctypes.c_ulong(buf.nbytes)
+            ptr = ctypes.byref(ctypes.c_void_p.from_buffer(buf))
```

Comment:
Hmm, that works too, but didn't solve the segfaults I see. Am I misunderstanding this?

https://docs.python.org/2/library/ctypes.html#ctypes.byref